### PR TITLE
11776 - CompanyCard Edits

### PR DESCRIPTION
**Pull request by andreferreiradlw**
TP11776

This PR implements some final edits and copy changes to the `CompanyCard` component as described by the ticket.

The current `Tooltips` found on this component now wrap the anchor text instead of displaying the _i_ icon.

Also removes the regex being used previously as this is causing some functionality to break in some browsers, this needs to be re-implemented on a later ticket.

### CODE REVIEW

File: src/components/CompanyCard/locale_en.json [hunk 1]
```diff
@@ -9,49 +9,50 @@
...
     },
     "coronavirusCancellationCover": {
       "title": "Coronavirus cover if trip cancelled",
-      "tooltip": "Not all policies will provide cancellation cover if the cancellation is linked to Coronavirus, but some will.\nFirms that have answered 'yes' to this question should offer cover whether or not you have the virus yourself or you have to self-isolate because you have been in contact with someone else who has the virus.\nEven if this cover is included, check the terms and conditions relating to cancellation carefully before you buy."
+      "tooltip": "Some policies will refund the cost of your trip if you have to cancel because of Coronavirus, but ONLY if the policyholder tests positive for Coronavirus and is unable to travel.\nIf the policyholder can’t travel because they are self-isolating or in quarantine, most policies won’t pay out.\nCheck all the policy conditions with your provider carefully before you buy."
     },
     "medicalEquipmentCover": {
       "title": "Medical equipment cover",
-      "offered": "Yes up to £",
+      "offered": "Yes: up to £",
```
Comment (davidtrussler) [inline src/components/CompanyCard/locale_en.json:44]: This should be updated to a comma `,` not colon `:`.

File: src/components/CompanyCard/locale_cy.json [hunk 2]
```diff
@@ -36,22 +37,22 @@
     },
     "coronavirusCancellationCover": {
       "title": "Yswiriant Coronafirws os bydd y daith yn cael ei chanslo",
-      "tooltip": "Ni fydd pob polisi yn darparu yswiriant canslo os yw'r canslo'n gysylltiedig â Coronavirus, ond bydd rhai yn gwneud hynny.\nDylai cwmnïau sydd wedi ateb 'ydw' i'r cwestiwn hwn gynnig yswiriant p'un a oes gennych y firws eich hun ai peidio neu a oes rhaid i chi hunan-ynysu oherwydd eich bod wedi bod mewn cysylltiad â rhywun arall sydd â'r firws.\nHyd yn oed os yw'r gorchudd hwn wedi'i gynnwys, gwiriwch y telerau ac amodau sy'n ymwneud â chanslo yn ofalus cyn i chi brynu."
+      "tooltip": "Bydd rhai polisïau yn ad-dalu cost eich taith os bydd yn rhaid i chi ganslo oherwydd Coronavirus, ond YN UNIG os yw'r deiliad polisi'n profi'n bositif am Coronavirus ac yn methu â theithio.\nOs na all deiliad y polisi deithio oherwydd ei fod yn hunan-ynysu neu mewn cwarantîn, ni fydd y mwyafrif o bolisïau yn talu allan.\nGwiriwch yr holl amodau polisi gyda'ch darparwr yn ofalus cyn i chi brynu"
     },
     "medicalEquipmentCover": {
       "title": "Gorchudd offer meddygol",
-      "offered": "Ie hyd at £",
+      "offered": "Ie: hyd at £",
```
Comment (davidtrussler) [inline src/components/CompanyCard/locale_cy.json:44]: This should be updated to a comma `,` not colon `:`.

File: src/components/CompanyCard/index.js [hunk 3]
```diff
@@ -163,22 +168,20 @@ const CompanyCard = ({
...
           {medical_equipment_cover && (
             <Info>
               <InfoTitle>{`${lng.moreInfo.medicalEquipmentCover.title} - `}</InfoTitle>
               {medical_equipment_cover.offers_cover
-                ? `${
-                    lng.moreInfo.medicalEquipmentCover.offered
-                  }${medical_equipment_cover.cover_amount
-                    .toString()
-                    .replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',')}`
+                ? `${lng.moreInfo.medicalEquipmentCover.offered} ${medical_equipment_cover.cover_amount}`
```
Comment (davidtrussler) [inline src/components/CompanyCard/index.js:184]: Could we remove the space here or put the `£` sign into the `cover_amount` text in order to keep the currency and amount together? 

image

**Comment by davidtrussler** [comment]
I've left a few small comments but I notice the TP Card often doesn't have changes for the Welsh copy equivalent to those in the English. Just want to be confident that this is not going to be overlooked.

**Comment by andreferreiradlw** [comment]
> I've left a few small comments but I notice the TP Card often doesn't have changes for the Welsh copy equivalent to those in the English. Just want to be confident that this is not going to be overlooked.

The Welsh translations in this PR are only temporary, the final Welsh translations are now being sourced by Gavin and should be implemented early next week.

**Comment by davidtrussler** [comment]
> The Welsh translations in this PR are only temporary, the final Welsh translations are now being sourced by Gavin and should be implemented early next week.



> > I've left a few small comments but I notice the TP Card often doesn't have changes for the Welsh copy equivalent to those in the English. Just want to be confident that this is not going to be overlooked.
> 
> The Welsh translations in this PR are only temporary, the final Welsh translations are now being sourced by Gavin and should be implemented early next week.

👍

**Review by davidtrussler — APPROVED** [review]
LGTM! 👍

**Review by akenzua — APPROVED** [review]
LGTM

**Pull request merged**